crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -1383,223 +1389,236 @@ async fn resolve_internal_inline(
     request: Vc<Request>,
     options: Vc<ResolveOptions>,
 ) -> Result<Vc<ResolveResult>> {
-    // This explicit deref of `options` is necessary
-    #[allow(clippy::explicit_auto_deref)]
-    let options_value: &ResolveOptions = &*options.await?;
-
-    // Apply import mappings if provided
-    if let Some(import_map) = &options_value.import_map {
-        let result = import_map.await?.lookup(lookup_path, request).await?;
-        if !matches!(result, ImportMapResult::NoEntry) {
-            let resolved_result = resolve_import_map_result(
-                &result,
-                lookup_path,
-                lookup_path,
-                request,
-                options,
-                request.query(),
-            )
-            .await?;
-            // We might have matched an alias in the import map, but there is no guarantee
-            // the alias actually resolves to something. For instance, a tsconfig.json
-            // `compilerOptions.paths` option might alias "@*" to "./*", which
-            // would also match a request to "@emotion/core". Here, we follow what the
-            // Typescript resolution algorithm does in case an alias match
-            // doesn't resolve to anything: fall back to resolving the request normally.
-            if let Some(result) = resolved_result {
-                return Ok(result);
+    let span = {
+        let lookup_path = lookup_path.to_string().await?;
+        let request = request.to_string().await?;
+        tracing::info_span!(
+            "internal resolving",
+            lookup_path = *lookup_path,
+            request = *request
+        )
+    };
+    async move {
```

Comment:
not a fan of adding indent to the whole function for tracing

Reply:
me neither, but `.enter()` doesn't work in async code...

Reply:
would rather have another function :^)